### PR TITLE
Fix: Update index.md about bookinfo in Chinese version

### PR DESCRIPTION
**Pull request by fengzixu**

### CODE REVIEW

**Comment by fengzixu** [comment]
@gyliu513 Request review

**Comment by fengzixu** [comment]
@rootsongjc

File: content_zh/docs/examples/bookinfo/index.md [hunk 1]
```diff
@@ -183,7 +183,7 @@ $ curl -o /dev/null -s -w "%{http_code}\n" http://${GATEWAY_URL}/productpage
 
 在使用 Istio 控制 Bookinfo 版本路由之前，你需要在目标规则中定义好可用的版本，命名为 *subsets* 。
 
-运行一下命令为 Bookinfo 服务创建的默认的目标规则：
+运行以下命令为 Bookinfo 服务创建的默认的目标规则：
 
 * 如果不需要启用 mutual TLS ，请执行以下命令：
```
Comment (rootsongjc) [inline content_zh/docs/examples/bookinfo/index.md]: mutual TLS 翻译成“双向TLS”

**Comment by rootsongjc** [comment]
CI failed.
```
- ./public/zh/about/contribute/writing-a-new-topic/index.html
  *  External link https://gohugo.io/content-management/front-matter/ failed: response code 0 means something's wrong.
             It's possible libcurl couldn't connect to the server or perhaps the request timed out.
             Sometimes, making too many requests at once also breaks things.
             Either way, the return message (if any) from the server is: Couldn't resolve host name
- ./public/zh/docs/concepts/performance-and-scalability/index.html
  *  External link https://github.com/istio/fortio/blob/master/README.md#fortio failed: 502 No error
- ./public/zh/docs/tasks/traffic-management/egress/index.html
  *  External link http://httpbin.org failed: 503 No error
htmlproofer 3.9.2 | Error:  HTML-Proofer found 3 failures!
LINTING FAILED
Exited with code 1
```
Please re-lint.

**Comment by fengzixu** [comment]
@rootsongjc I think that you should revise CI of this project. Most of PRs have already failed and the info of error illustrates that the network perhaps gets an error.

**Comment by fengzixu** [comment]
@rootsongjc Fixed.

File: content_zh/docs/examples/bookinfo/index.md [hunk 2]
```diff
@@ -185,7 +185,7 @@ $ curl -o /dev/null -s -w "%{http_code}\n" http://${GATEWAY_URL}/productpage
 
 运行以下命令为 Bookinfo 服务创建的默认的目标规则：
 
-* 如果不需要启用 mutual TLS ，请执行以下命令：
+* 如果不需要启用双向TLS ，请执行以下命令：
```
Comment (gyliu513) [inline content_zh/docs/examples/bookinfo/index.md]: L194 also need update.

File: content_zh/docs/examples/bookinfo/index.md [hunk 3]
```diff
@@ -183,15 +183,15 @@ $ curl -o /dev/null -s -w "%{http_code}\n" http://${GATEWAY_URL}/productpage
...
 
     {{< text bash >}}
     $ kubectl apply -f @samples/bookinfo/networking/destination-rule-all.yaml@
     {{< /text >}}
 
-* 如果需要启用 mutual TLS ，请执行以下命令：
+* 如果需要启用双向 TLS ，请执行以下命令：
```
Comment (gyliu513) [inline content_zh/docs/examples/bookinfo/index.md]: `双向 TLS` 需要空格吗

**Pull request merged**